odata-csdl/4 CSDL Document.md

```diff
@@ -125,10 +125,10 @@ Example ##ex: references to other CSDL documents
 {
   ...
   "$Reference": {
-    "http://vocabs.odata.org/capabilities/v1": {
+    "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Capabilities.V1.xml": {
```

Comment:
Use the `.json` files within the JSON representation, or keep referencing the XML to drive home that one can "mix" representations?